3.5-windowsservercore-1709/build/Dockerfile

```diff
@@ -1,36 +1,38 @@
+# escape=`
+
 FROM microsoft/dotnet-framework:3.5-windowsservercore-1709
 
 SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
 
 # Install NuGet CLI
 ENV NUGET_VERSION 4.4.1
-RUN New-Item -Type Directory $Env:ProgramFiles\NuGet; \
+RUN New-Item -Type Directory $Env:ProgramFiles\NuGet; `
```

Comment:
@natemcmaster @glennc Do we need something for ASP.NET images as well?

Reply:
I believe the ASP.NET Core Windows Dockerfiles are already using the backtick escape - https://github.com/aspnet/aspnet-docker/blob/master/1.1/nanoserver-sac2016/sdk/Dockerfile#L1

Reply:
Correct, we already do this.